.travis.yml

```diff
@@ -1,7 +1,8 @@
 sudo: true
 language: go
 go:
-  - 1.11.x
+  - 1.12.x
+  - 1.13.x
```

Comment:
To avoid that in the future I think we could just use `tip`.

Reply:
That's a good idea. In Travis CI docs they say https://github.com/travis-ci/gimme is used to get the Go version, so we could use `stable` and `oldstable` instead of specific versions.